prow/images/cleaner/cleaner.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+echo "Authenticating to Google Cloud..."
+export CLOUDSDK_COMPUTE_ZONE="europe-west3-a"
+export CLOUDSDK_COMPUTE_REGION="europe-west3"
```

Comment:
It should be configurable. Define it in preset, or use preset from this PR: https://github.com/kyma-project/test-infra/pull/98/files

Reply:
agree, after #98 will be merged, I will update my PR.